ansible/inventories/devnet-0/group_vars/bootnode.yaml

```diff
@@ -13,6 +13,7 @@ eth_inventory_web_container_env:
 ethereum_node_el: geth
 ethereum_node_cl: teku
 ethereum_node_cl_validator_enabled: false
+ethereum_node_cl_checkpoint_sync_enabled: "{{ checkpoint_sync_enabled }}"
```

Comment:
should be explicitly set to false for the bootnode, the bootnode can't checkpoint sync off of anything in a template testnet in case the user by mistake sets it to `true`

Reply:
Yeah this variable shouldn't be present anymore actually. Thanks for catching it.